src/apps/frontend/services/account.service.ts

```diff
@@ -15,4 +15,12 @@ export default class AccountService extends APIService {
       },
     });
   };
+
+  deleteAccount = async (): Promise<ApiResponse<void>> => {
+    const userAccessToken = JSON.parse(
```

Comment:
Think this should be a common logic defined which can be reused in our code. Ideally - This should be within the same code where the token is being saved in storage in the first place.

Reply:
Made a common logic for accessing, setting and removing access token from storage